### PR TITLE
Update plot_DroppedData_table.R

### PR DESCRIPTION
**Pull request by QLLZ**
updated drop data summary table in country data report

### CODE REVIEW

File: packages/taxdat/R/plot_DroppedData_table.R [hunk 1]
```diff
@@ -52,6 +52,20 @@ plot_DroppedData_table <- function(config, cache, cholera_directory, aesthetic =
   
   all_obs_stats <- all_obs_stats[order(all_obs_stats$year), ]
   
+  #when all observations in a certain year are dropped:
+  if(nrow(used_obs_stats) < nrow(all_obs_stats)){ 
+    dropped_year <- all_obs_stats$year[!all_obs_stats$year%in%used_obs_stats$year&!all_obs_stats$year=='all']
+    used_obs_stats<-used_obs_stats%>%
+      add_row(year = dropped_year,
```
Comment (eclee25) [inline packages/taxdat/R/plot_DroppedData_table.R:59]: Can you specify the package names here and line 66?

File: packages/taxdat/R/plot_DroppedData_table.R [hunk 2]
```diff
@@ -33,7 +33,7 @@ plot_DroppedData_table <- function(config, cache, cholera_directory, aesthetic =
   used_obs_stats <- obs_stats[order(obs_stats$year), ] %>% 
     dplyr::filter(!grepl('imputed',u_OCs))
   #get all obs_stats
-  get_stan_input(name="stan_input",config=config, cache=cache, cholera_directory=cholera_directory)
+  get_sf_cases(name="sf_cases",config=config, cache=cache, cholera_directory=cholera_directory)
```
Comment (eclee25) [inline packages/taxdat/R/plot_DroppedData_table.R:36]: I don't understand how this function worked before if we had not loaded sf_cases into cache[["sf_cases"]]...

**Review by eclee25 — CHANGES_REQUESTED** [review]

File: packages/taxdat/R/plot_DroppedData_table.R (hunk 2 above)
Comment (QLLZ) [reply packages/taxdat/R/plot_DroppedData_table.R:36]: I think it was loaded outside the package in the Rmarkdown file (here: https://github.com/HopkinsIDD/cholera-mapping-pipeline/blob/dev/Analysis/output/country_data_report.Rmd#L131)

**Comment by QLLZ** [comment]
I just added the moz case raster plot to taxdat function too.

File: packages/taxdat/R/plot_case_raster_admin1_shp.R [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#' @export
+#' @name plot_case_raster_admin_shp
+#' @param config
+#' @param cholera_directory
+#' @param cache
+plot_case_raster_admin_shp <- function(cache, cholera_directory, config) {
```
Comment (eclee25) [inline packages/taxdat/R/plot_case_raster_admin1_shp.R:6]: I think we should make the function name match the name of the R script. Could you align these and rerun devtools::document on the package?

**Review by eclee25 — CHANGES_REQUESTED** [review]

**Pull request merged**